package.json

```diff
@@ -12,7 +12,7 @@
   },
   "scripts": {
     "prebuild": "npm run clean",
-    "build": "npm-run-all -p build:*",
```

Comment:
This pattern runs `build:js` and `build:test` in parallel, it will ignore the third level
See https://github.com/mysticatea/npm-run-all/blob/master/docs/npm-run-all.md#glob-like-pattern-matching-for-script-names

Reply:
Ok in my setup it's a little different I notice now, and you're right so ignore this change.